proto/zitadel/session/v2/session.proto

```diff
@@ -157,9 +159,31 @@ message CreationDateQuery {
   ];
 }
 
+message CreatorQuery {
+  // ID of the user used to create the session, if empty the id of the calling user is used as creator.
+  optional string id = 1 [
+    (validate.rules).string = {max_len: 200},
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      max_length: 200;
+      example: "\"69629023906488334\"";
+    }
+  ];
+}
+
+message UserAgentQuery {
+  // ID of the useragent used for the session, if empty the used useragent for the call is used.
+  optional string id = 1 [
```

Comment:
I've looked at this again and would make it clearer incl. naming it fingerprint_id to match the name in the `UserAgent` message:
```suggestion
  // Finger print id of the user agent used for the session.
  // Set an empty fingerprint_id to use the user agent from the call.
  // If the user agent is not available from the current token, an error will be returned.
  optional string fingerprint_id = 1 [
```